scib/preprocessing.py

```diff
@@ -774,7 +788,9 @@ def read_conos(inPath, dir_path=None):
     """
     from shutil import rmtree
 
-    import pandas as pd
+    import rpy2.rinterface_lib.callbacks
+    import rpy2.rinterface_lib.embedded
+    import rpy2.robjects as ro
```

Comment:
why are the rpy2 import statements not wrapped in a try-catch block in this file?